core/templates/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.directive.html

```diff
@@ -33,6 +33,27 @@
     -o-transition: all 0.15s;
     transition: all 0.15s;
   }
+  .oppia-graph-viz .oppia-material-icons {
```

Comment:
Take a look at where oppia-graph-viz is used. Does it enclose oppia-material-icons? Please be careful.